### ISSUE TITLE
The JournalEntry Example breaks in Visual Studio Emulator for Android

### ISSUE THREAD

**Issue by HobDev** [realm/realm-dotnet#1363]
## Goals

Run the sample

## Expected Results

Run successfully

## Actual Results

breaks when Save Button is clicked

## Steps to Reproduce

Run on Visual Studio Emulator for Android(Android version  6 - Api 23)

## Code Sample

_A code sample was not provided by @HobDev._

## Version of Realm and Tooling

Realm version 1.2.0, Visual studio 2017 

![screenshot 2](https://cloud.githubusercontent.com/assets/9531864/25306023/5d532d00-27a3-11e7-8a89-49f8f78eae92.png)

- Realm Object Server Version: ?
- Flavor: Developer
- Server OS & Version: ? (e.g. CentOS 6)
- Client SDK Version: ?
- Client OS & Version: Windows 10 Creators update

**Comment by HobDev** [comment]
The screen shot of the visual studio feedback on application breakdown is attached with the issue.

**Comment by HobDev** [comment]
The sample runs fine on my Android device. The device (LG Nexus 5X) is running latest android version 7.1,  api level 25.

**Comment by istx25** [comment]
Hi @HobDev. Thanks for reaching out. Someone will follow-up with you soon.

**Comment by nirinchev** [comment]
Sorry for the delay. Can you let us know what exception is being thrown?

**Comment by HobDev** [comment]
I updated visual studio on 11/05/2017. I also updated fody to 2.0.7 , newtonsoft and http client packages to latest on 11/05/17. Now the project didn't build. There were four error messages.

-The "PackageDefinitions" parameter is not supported by the "Fody.WeavingTask" task. Verify the parameter exists on the task, and it is a settable public instance property.	QuickJournal		

-The "Fody.WeavingTask" task could not be initialized with its input parameters.	QuickJournal		

-The "PackageDefinitions" parameter is not supported by the "Fody.WeavingTask" task. Verify the parameter exists on the task, and it is a settable public instance property.	QuickJournal.Droid		

-The "Fody.WeavingTask" task could not be initialized with its input parameters.	QuickJournal.Droid		

The actual problem changed.

**Comment by nirinchev** [comment]
Can you try to update the Realm version as well? Seems like you're not using the latest one.

**Comment by HobDev** [comment]
I am using the latest 1.2.1

**Comment by nirinchev** [comment]
It seems you have a leftover nugets in the packages folder. You can either remove those or manually edit the `.csproj` files to remove this line: 
```
<Import Project="..\packages\Fody.1.29.4\build\portable-net+sl+win+wpa+wp\Fody.targets" Condition="Exists('..\packages\Fody.1.29.4\build\portable-net+sl+win+wpa+wp\Fody.targets')" />
```

**Comment by HobDev** [comment]
I edited the .csproj file and the project now builds. Now the actual issue arises. When I depoly to the device the app works fine. But when i deploy to visual studio emulator for android (api level 5.1.1) it through an exception when i press save button to save the entry. The exception says: "An unhandled exception occured".

**Comment by HobDev** [comment]
When I press `Add` button the navigation happens and I able to write the entry. But the exception raises when I press `save` button.

**Comment by HobDev** [comment]
The issue is resolved with new realm version 1.3.0. There was some bug in version 1.2.1. So, closing the issue.
